### ISSUE TITLE
Uknown Error while viewing datadrame

### ISSUE THREAD

**Issue by MutuaNdunda** [spyder-ide/spyder#17312]
### What steps will reproduce the problem?

#Hello check out this error it appeared when I viewed dataframe in the enviroment pane.

### Traceback
```python-traceback
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
TypeError: invalid result from DataFrameHeaderModel.headerData(), unable to convert a Python 'Alignment' object to a C++ 'Qt::Alignment' instance
```

## Versions

* Spyder version: 5.2.2 
* Python version: 3.10.0
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Linux 5.14.17-301.fc35.x86_64

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0;<8.0.0        :  7.30.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jellyfish >=0.7               :  0.8.9 (OK)
jsonschema >=3.2.0            :  4.3.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.3.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=2.5.0                :  2.12.2 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.0.1 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=1.0.2             :  1.1.1 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  1.11.3 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  59.6.0 (OK)
sphinx >=0.6.6                :  4.3.1 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.2.2 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
xdg >=0.26                    :  0.27 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  None (NOK)
numpy >=1.7                   :  1.21.4 (OK)
pandas >=1.1.1                :  1.3.5 (OK)
scipy >=0.17.0                :  None (NOK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by ccordoba12** [comment]
Hey @MutuaNdunda, thanks for reporting. I can't reproduce this error using Python 3.10.2 and PyQt 5.15.

How did you install Spyder?

**Comment by ccordoba12** [comment]
Closing due to lack of response.

**Comment by jacksund** [comment]
@ccordoba12 I'm having the same issue, so this may be worth reopening.

I installed spyder from the conda-forge channel with PyQt 5.12.3 and Python 3.10.2. Maybe the issue can be fixed with PyQt 5.15, but it looks like that release won't be available any time soon (per [this issue](https://github.com/conda-forge/pyqt-feedstock/issues/98)).

**Comment by ccordoba12** [comment]
See issue #17361.
